pontoon/base/views.py

```diff
@@ -1313,10 +1313,6 @@ def request_locale(request):
 
 class BrowserIDVerify(BrowserIDVerifyBase):
     def login_success(self):
-        # Check for permission to localize if not granted on every login
-        if not self.user.has_perm('base.can_localize'):
-            utils.add_can_localize(self.user)
-
```

Comment:
If you're removing this you might as well get rid of the whole class and use the default verifier.